index.rst

```diff
@@ -145,16 +154,24 @@ An example report generated by Robot Framework is available `here <_static/tts_c
 
 .. _Robot Framework: https://robotframework.org
 
+One of the key aspects of requirement verification is to be able to map a particular test with a particular requirement it verifies.
+For that, Robot framework allows us to add tags to test reports.
+Using those tags we can assign the list of requirements that are verified by each individual test.
+
 .. _activity-sequence-diagrams:
 
 Activity Sequence Diagrams
 --------------------------
 
+Most of the software requirement verification we perform consist of demonstrating that a particular component executes a particular set of actions.
+These are mostly represented by a set of events, telemetry or event commands published by the component after receiving a particular command or as the by-product of a particular event/telemetry published by another component.
```

Comment:
event commands? Consider:
The success criteria are often represented by the publishing of specific sequence or set of events or telemetry.
In many cases, success is evaluated by observing events and their payloads that are published in response to a command sent to the CSC, or that are published in response to information published by another component.